example/non_rails_app/spec/lib/non_rails_app/event_handler_spec.rb

```diff
@@ -6,7 +6,7 @@ module NonRailsApp
     describe '#handle_event' do
       it 'publishes an update event' do
         expect{
-          described_class.handle_event double(:event, name: 'user_updates_beaver', payload: { 'beaver_name' => 'beaver' })
+          described_class.handle_event RabbitFeed::Event.new({'name' => 'user_updates_beaver', 'application' => 'rails_app'}, {'beaver_name' => 'beaver'})
```

Comment:
Good question. For testing, the minimum that is required is the `name` field. What are your thoughts?